bohye/board-project/src/App.js

```diff
@@ -2,6 +2,17 @@ import React, { useState, useEffect } from "react";
 import Board from "./components/Board";
 import "./App.css";
 
+const fetchData = async () => {
+  try {
+    const response = await fetch("http://heojh.iptime.org:8003/board");
+    const data = await response.json();
+    return data;
```

Comment:
data 변수를 다음 코드에서 사용하지 않으므로 다음과 같이 수정하면 좋을 것 같습니다.
```suggestion
    return await response.json();
```